recipes/libyuv/all/conandata.yml

```
@@ -0,0 +1,8 @@
+sources:
+  "20201106":
```

```suggestion
  "cci.20201106":
```

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

---

fixed